storage/aptosdb/src/db_options.rs

```diff
@@ -102,6 +102,14 @@ pub(super) fn state_merkle_db_column_families() -> Vec<ColumnFamilyName> {
     ]
 }
 
+pub(super) fn state_merkle_db_metric_reporting_column_families() -> Vec<ColumnFamilyName> {
```

Comment:
seems cleaner to filter out the DEFAULT (for all DBs) AND DB_METADATA (for db shards) cf names on the call side?